### PR TITLE
fix(web): tag cloud example

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
the demo was breaking here: https://codesandbox.io/s/github/appbaseio/reactivesearch/tree/next/packages/web/examples/TagCloud?from-embed

### CODE REVIEW

File: packages/web/examples/TagCloud/src/index.js [hunk 1]
```diff
@@ -61,16 +64,17 @@ const Main = () => (
...
-														.map(tag => (
-															<div
-																className="meetup-topic"
-																key={tag.topic_name}
-															>
-																{tag.topic_name}
-															</div>
-														))}
+													{data.group
+														&& data.group.group_topics
```
Comment (jyash97) [inline packages/web/examples/TagCloud/src/index.js]: `data.group && data.group.group_topics &&  ...`

Comment (ShahAnuj2610) [reply packages/web/examples/TagCloud/src/index.js]: It's `data.group && data.group.group_topics.slice`

Comment (jyash97) [reply packages/web/examples/TagCloud/src/index.js]: We can add check for `groupd_topics` as well. Just to be on safe side

Comment (ShahAnuj2610) [reply packages/web/examples/TagCloud/src/index.js]: done

**Review by jyash97 — APPROVED** [review]

**Pull request merged**